### PR TITLE
Adding `quera` backend

### PR DESCRIPTION
**Pull request by khalatepradnya**
* Skeleton for `quera` target based on Braket
* Frontend will be added in separate PR.

### CODE REVIEW

File: python/runtime/cudaq/platform/py_alt_launch_kernel.cpp [hunk 1]
```diff
@@ -413,6 +413,16 @@ void pyAltLaunchKernel(const std::string &name, MlirModule module,
...
 }
 
+void pyAltLaunchAnalogKernel(const std::string &name,
+                             std::string &programArgs) {
+  // TODO: check name
+  auto dynamicResult = cudaq::altLaunchKernel(
+      name.c_str(), KernelThunkType(nullptr),
+      (void *)(const_cast<char *>(programArgs.c_str())), 0, 0);
+  if (dynamicResult.data_buffer || dynamicResult.size)
+    throw std::runtime_error("not implemented: support dynamic results");
```
[the remark this answers is not in the document]
Comment (khalatepradnya) [reply python/runtime/cudaq/platform/py_alt_launch_kernel.cpp]: Fixed in b9882f2

File: python/runtime/cudaq/platform/py_alt_launch_kernel.cpp [hunk 2]
```diff
@@ -413,6 +413,16 @@ void pyAltLaunchKernel(const std::string &name, MlirModule module,
   std::free(rawArgs);
 }
 
+void pyAltLaunchAnalogKernel(const std::string &name,
+                             std::string &programArgs) {
+  // TODO: check name
```
Comment (bettinaheim) [inline python/runtime/cudaq/platform/py_alt_launch_kernel.cpp:418]: What name?

File: python/runtime/cudaq/platform/py_alt_launch_kernel.cpp [hunk 3]
```diff
@@ -683,6 +693,13 @@ void bindAltLaunchKernel(py::module &mod) {
       py::kw_only(), py::arg("callable_names") = std::vector<std::string>{},
       "DOC STRING");
 
+  mod.def(
+      "pyAltLaunchAnalogKernel",
+      [&](const std::string &name, std::string &programArgs) {
+        return pyAltLaunchAnalogKernel(name, programArgs);
+      },
+      py::arg("name"), py::arg("programArgs"), "DOC STRING");
```
Comment (bettinaheim) [inline python/runtime/cudaq/platform/py_alt_launch_kernel.cpp:701]: DOC STRING? :)

**Review by bettinaheim — APPROVED** [review]
Lgtm, but I may not spot issues with the launch/qpu.

File: python/runtime/cudaq/platform/py_alt_launch_kernel.cpp (hunk 2 above)
Comment (khalatepradnya) [reply python/runtime/cudaq/platform/py_alt_launch_kernel.cpp:418]: This is TODO for frontend merge where the kernel name will be fixed prefix and random string, for example,  `__analog_hamiltonian_kernel__xxxxxx`

**Pull request merged**